@AH-64D Apache Longbow/Addons/fza_ah64_customise/config/CfgAce.hpp

```diff
@@ -366,4 +366,38 @@ class ACE_Actions : ACE_Actions {
         selection   = "pilot_action";
         posiiton    = "[3,2.1,1.5]";
     };
+    class ACE_MainActions: ACE_MainActions {
```

Comment:
If we were to do that, I would prefer to have a siphon fuel which you can then set a percentage for